src/main/resources/assets/systeams/lang/zh_cn.json

```diff
@@ -1,13 +1,22 @@
 {
   "item.systeams.steam_bucket": "蒸汽桶",
+  "item.systeams.steamier_bucket": "温热蒸汽桶",
+  "item.systeams.steamiest_bucket": "高温蒸汽桶",
+  "item.systeams.steamiester_bucket": "超高温蒸汽桶",
+  "item.systeams.steamiestest_ball": "等离子球",
   "fluid_type.systeams.steam": "蒸汽",
+  "fluid_type.systeams.steamier": "温热蒸汽",
+  "fluid_type.systeams.steamiest": "高温蒸汽",
+  "fluid_type.systeams.steamiester": "超高温蒸汽",
+  "fluid_type.systeams.steamiestest": "等离子体",
   "item.systeams.boiler_pipe": "蒸汽管道",
   "item.systeams.boiler_pipe.keyword": "蒸汽锅炉线圈",
   "item.thermal.rf_coil.keyword": "蒸汽锅炉发电机",
-  "item.systeams.boiler_pipe.desc": "潜行右击能源炉将其转换为锅炉，返回另一个头部零件。",
-  "item.thermal.rf_coil.desc": "潜行右击锅炉将其转换为能源炉，返回蒸汽管道。",
+  "item.systeams.boiler_pipe.desc": "潜行右击能源炉将其转换为锅炉，返还红石通量线圈。",
```

Comment:
[The English version](https://github.com/ChiefArug/ThermalSysteams/blob/e1dc244f63b098bef869cab3641767d64704cb4f/src/main/resources/assets/systeams/lang/en_us.json#L15) for this very specifically doesn't mention the Redstone Flux Coil (红石通量线圈) and is a bit vague about what is returned because when using it on a pneumatic dynamo a different item is returned.